### PR TITLE
feat: updated pages

### PR DESCRIPTION
**Pull request by rishavnaskar**
# Pull Request Type
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Purpose
- WHAT is this PR for?

# Approach
- HOW does this change fulfill the purpose?

# Does this PR introduce any new bug?
- [ ] Yes
- [x] No
- [ ] Maybe

#### Issue Reference: paste the issue link here if any or else remove this line

# Other information

# Did you learn something new?
- Describe what did you learn

### CODE REVIEW

**Comment by BRO3886** [comment]
@rishavnaskar should I merge?

**Comment by rishavnaskar** [comment]
> @rishavnaskar should I merge?

Yes please

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -56,4 +56,8 @@ jobs:
           VALIDATE_DOCKERFILE: false
           VALIDATE_DOCKERFILE_HADOLINT: false
           VALIDATE_PYTHON_FLAKE8: false
+          VALIDATE_JSCPD: false
+          VALIDATE_KOTLIN: false
```
Comment (BRO3886) [inline .github/workflows/linter.yml]: Can you set kotlin as true?

File: app/google-services.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (BRO3886) [inline app/google-services.json:1]: Uhm file should be in gitignore 😳

**Review by BRO3886 — CHANGES_REQUESTED** [review]

File: app/google-services.json (hunk 2 above)
Comment (rishavnaskar) [reply app/google-services.json:1]: Cool okay, will do that. But I think this file doesn't have any confidential server APIs.

File: .github/workflows/linter.yml (hunk 1 above)
Comment (rishavnaskar) [reply .github/workflows/linter.yml]: Okay. I was just afraid of getting errors.

**Pull request merged**